src/components/Questionnaire/QuestionTypes/DeathQuestion.tsx

```diff
@@ -0,0 +1,47 @@
+import { DateTimePicker } from "@/components/ui/date-time-picker";
+
+import { QuestionValidationError } from "@/types/questionnaire/batch";
+import {
+  QuestionnaireResponse,
+  ResponseValue,
+} from "@/types/questionnaire/form";
+import { Question } from "@/types/questionnaire/question";
+
+interface TimeOfDeathQuestionProps {
+  question: Question;
+  questionnaireResponse: QuestionnaireResponse;
+  updateQuestionnaireResponseCB: (
+    values: ResponseValue[],
+    questionId: string,
+    note?: string,
+  ) => void;
+  disabled?: boolean;
+  errors: QuestionValidationError[];
```

Comment:
we are not showing QuesiontValidationError for this question anywhere here.

Reply:
Removed, the default validation is enough.